add-ons/ansible-service-broker/README.md

```diff
@@ -0,0 +1,71 @@
+# Ansible Service Broker Add-on
+
+
+This addon installs the [Ansible Service Broker](https://github.com/openshift/ansible-service-broker)
+
+## Prerequisites
+
+Minishift/CDK must be deployed with the Service Catalog enabled
+
+The Service Catalog can be deployed by adding the `--service-catalog` flag when starting Minishift. The ability to pass extra flags during startup can be enabled by setting the _MINISHIFT_ENABLE_EXPERIMENTAL_ environmental variable as follows:
+
+```
+$ export MINISHIFT_ENABLE_EXPERIMENTAL=y
+```
+
+Start Minishift with the `--service-catalog` flag:
+
+```
+$ minishift start --service-catalog
+```
```

Comment:
@sabre1041 I think OpenShift v3.7.0 is advisable for this add-on as 3.6.0 had different implementation for `oc cluster up` and it did not work always. WDYT?

Reply:
@sabre1041 tested on v 3.7.0 and it worked for me. Going ahead and merging this, please send another PR if you want to set minimum version to v3.7.0 https://docs.openshift.org/latest/minishift/using/addons.html#addon-openshift-version-semantics
```
$ minishift start --openshift-version v3.7.0 --service-catalog
-- Starting profile 'minishift'
-- Checking if requested hypervisor 'kvm' is supported on this platform ... OK
-- Checking if KVM driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-kvm ... 
   Checking driver binary is executable ... OK
-- Checking if Libvirt is installed ... OK
-- Checking if Libvirt default network is present ... OK
-- Checking if Libvirt default network is active ... OK
-- Checking the ISO URL ... OK
-- Starting local OpenShift cluster using 'kvm' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB
-- Starting Minishift VM ...................... OK
-- Checking for IP address ... OK
-- Checking if external host is reachable from the Minishift VM ... 
   Pinging 8.8.8.8 ... OK
-- Checking HTTP connectivity from the VM ... 
   Retrieving http://minishift.io/index.html ... OK
-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 0% used OK
-- Downloading OpenShift binary 'oc' version 'v3.7.0'
 38.51 MiB / 38.51 MiB [=========================================================================================================================================] 100.00% 0s-- Downloading OpenShift v3.7.0 checksums ... OK
-- OpenShift cluster will be configured with ...
   Version: v3.7.0
-- Checking `oc` support for startup flags ... 
   host-config-dir ... OK
   host-data-dir ... OK
   host-pv-dir ... OK
   host-volumes-dir ... OK
   version ... OK
   routing-suffix ... OK
   service-catalog ... OK
Starting OpenShift using openshift/origin:v3.7.0 ...
Pulling image openshift/origin:v3.7.0
Pulled 1/4 layers, 26% complete
Pulled 1/4 layers, 42% complete
Pulled 1/4 layers, 57% complete
Pulled 1/4 layers, 72% complete
Pulled 2/4 layers, 76% complete
Pulled 3/4 layers, 88% complete
Pulled 3/4 layers, 99% complete
Pulled 4/4 layers, 100% complete
Extracting
Image pull complete
OpenShift server started.

The server is accessible via web console at:
    https://192.168.42.113:8443

You are logged in as:
    User:     developer
    Password: <any value>

To login as administrator:
    oc login -u system:admin

[lmohanty@LalatenduM-laptop minishift-addons]$ minishift addon apply ansible-service-broker
-- Applying addon 'ansible-service-broker':.......
Ansible Service Broker Deployed
```